os-eap-probes/added/livenessProbe.sh

```diff
@@ -16,26 +16,60 @@ COUNT=30
 SLEEP=5
 DEBUG_SCRIPT=false
 PROBE_IMPL="probe.eap.dmr.EapProbe probe.eap.dmr.HealthCheckProbe"
+INITIAL_SLEEP_SECONDS=5
+TMPLOC=${TMPDIR:-/tmp}
 
 if [ $# -gt 0 ] ; then
     COUNT=$1
 fi
 
-if [ $# -gt 1 ] ; then
-    SLEEP=$2
-fi
+# setting the first parameter to true/false, gets you the new default behaviour of
+# count = 1, sleep = 0 and initial sleep seconds = 0
+# setting it to a numeric value, or nothing falls through to the old behaviour
 
-if [ $# -gt 2 ] ; then
-    DEBUG_SCRIPT=$3
-fi
+if [[ -n "$COUNT" && ( $COUNT = "true" || $COUNT = "false" ) ]]; then
+  COUNT=1
+  SLEEP=0
+  INITIAL_SLEEP_SECONDS=0
+
+  if [ $# -gt 1 ] ; then
+     DEBUG_SCRIPT=$2
+  fi
+
+  if [ $# -gt 2 ] ; then
+     PROBE_IMPL=$3
+  fi
+else
+    # deprecated support for count / sleep / initial sleep of 5s
+    if [ ! -f ${TMPLOC}/probe_deprecation_warning ]; then
```

Comment:
This should also test if $# -gt 0 as I think you'll get here with no params set at all.